cmake/flags.cmake

```diff
@@ -133,7 +133,6 @@ if (NOT WIN32)
 set(COMMON_FLAGS
     -fPIC
     -fno-omit-frame-pointer
-    -Werror
```

Comment:
is this removing of -Werror intentional?

Reply:
It's only temporary to silence paddle_bfloat compilation warnings, that won't be included into the final version of this PR